### ISSUE TITLE
Address Mongodb connection warnings

### ISSUE THREAD

**Issue by nathanielrindlaub** [tnc-ca-geo/animl-api#96]
We're getting the following connection warnings from MongoDB:

```
- DeprecationWarning: current URL string parser is deprecated, and will be removed in a future version. To use the new parser, pass option { useNewUrlParser: true } to MongoClient.connect.
- [MONGODB DRIVER] Warning: Top-level use of w, wtimeout, j, and fsync is deprecated. Use writeConcern instead.
- [MONGODB DRIVER] Warning: Current Server Discovery and Monitoring engine is deprecated, and will be removed in a future version. To use the new Server Discover and Monitoring engine, pass option { useUnifiedTopology: true } to the MongoClient constructor.
- DeprecationWarning: collection.ensureIndex is deprecated. Use createIndexes instead.
```

**Comment by ingalls** [comment]
@nathanielrindlaub Looked up most of those - we should be good for now as long as we don't jump to the next major Mongoose version. I spent a couple hours during this phase getting us off mongoose@5 and onto the latest mongoose@7. I fixed all the major breaking changes between versions but in an effort to reduce far-reaching changes in my already PR I held off on changing to new behaviours if the config options weren't removed/retired. We should be able to update the codebase to be fully compliant with the latest version (using no deprecated features) in a subsequent phase.
